client/src/components/heatmap/Day.tsx

```diff
@@ -30,20 +30,20 @@ const DayHeatmap = ({ data, stats, dayOnClick }: DayHeatmapProps) => {
   let cellColor: string;
   if (numberOfSongsPlayed === 0) {
     cellColor = 'bg-gray-100 border-gray-300 dark:bg-gray-700 dark:border-gray-500';
```

Comment:
change dark:border to `dark:border-gray-700`

Reply:
No border looks off...

Reply:
![image](https://user-images.githubusercontent.com/55932881/216690371-1d8b9b3c-ae6a-4748-9d08-2fdda0bcf2f7.png)
This is with `dark:bordder-gray-600`

Reply:
![image](https://user-images.githubusercontent.com/55932881/216690480-ab82dc5a-dd5c-40ab-9c36-118765c1647c.png)
This is with dark:bordder-gray-700

Reply:
you can choose

Reply:
I will pick 600.